src/moar.c

```diff
@@ -282,6 +282,13 @@ MVMInstance * MVM_vm_create_instance(void) {
             instance->coverage_log_fh = fopen_perhaps_with_pid(coverage_log, "a");
         else
             instance->coverage_log_fh = stderr;
+
+        instance->coverage_control = 0;
+        if (getenv("MVM_COVERAGE_CONTROL")) {
+            char *coverage_control = getenv("MVM_COVERAGE_CONTROL");
+            if (coverage_control && strlen(coverage_control))
```

Comment:
I just updated the rest of the file to remove unneeded strlen's https://github.com/MoarVM/MoarVM/commit/911b6323de2cc3f55b92268a88141669c8857d89

Change this line to
```c
if (coverage_control && coverage_control[0])
```